packages/application-shell/src/index.js

```diff
@@ -18,6 +18,10 @@ export {
 export { selectUserId, selectProjectKeyFromUrl } from './utils';
 export { default as AsyncChunkLoader } from './components/async-chunk-loader';
 export { GtmContext } from './components/gtm-booter';
+export {
+  default as GtmUserLogoutTracker,
```

Comment:
Should we move this to the auth app? It's not used anymore here.

Reply:
I thought about it but then we spread logic: set-up in app-kit and teardown in app. That's why I liked to just export it.

Reply:
Yeah I guess...